src/Apps/Artist/Routes/AuctionResults/AuctionResultsFilterContext.tsx

```diff
@@ -6,6 +6,13 @@ export interface AuctionResultsFilters {
   sizes?: string[]
   page?: number
   sort?: string
+  createdAfterYear?: string
+  createdBeforeYear?: string
+
+  /** Used to get the overall earliest created year for all lots of given artist */
+  readonly earliestCreatedYear?: number
+  /** Used to get the overall latest created year for all lots of given artist */
+  readonly latestCreatedYear?: number
```

Comment:
These are only set initially via the context provider and aren't actually intended to be changed later. They're just holding the _total_ range of years created. Needed to populate the selector w/ dates.